tests/test_balances.py

```diff
@@ -0,0 +1,170 @@
+        "type": "bank-account",
+        "beneficiaryName": "Jack Bauer",
+        "bankAccount": "NL53INGB0654422370",
+        "bankAccountId": "bnk_jrty3f",
+    }
+
+
+def test_get_primary_balance(client, response):
+    """Get the primary balance"""
+    response.get("https://api.mollie.com/v2/balances/primary", "balance_single")
+    balance = client.balances.get("primary")
+
+    assert isinstance(balance, Balance)
+
+    assert balance.resource == "balance"
+    assert balance.id == BALANCE_ID
+    assert balance.mode == "test"
+    assert balance.created_at == "2019-01-10T10:23:41+00:00"
+    assert balance.currency == "EUR"
+    assert balance.status == "active"
+    assert balance.available_amount == {"value": "905.25", "currency": "EUR"}
+    assert balance.pending_amount == {"value": "0.00", "currency": "EUR"}
+
+    assert balance.transfer_frequency == "twice-a-month"
+    assert balance.transfer_threshhold == {"value": "5.00", "currency": "EUR"}
+
+    assert balance.transfer_reference == "Mollie payout"
+    assert balance.transfer_destination == {
+        "type": "bank-account",
+        "beneficiaryName": "Jack Bauer",
+        "bankAccount": "NL53INGB0654422370",
+        "bankAccountId": "bnk_jrty3f",
+    }
+
+
+def test_get_balance_report(client, response):
+    """Get a balance report."""
+    response.get(f"https://api.mollie.com/v2/balances/{BALANCE_ID}", "balance_single")
+    response.get(f"https://api.mollie.com/v2/balances/{BALANCE_ID}/report", "balance_report_single")
+
+    balance = client.balances.get(BALANCE_ID)
+    balance_report = balance.report.get()
+    assert isinstance(balance_report, BalanceReport)
+
+    assert balance_report.resource == "balance-report"
+    assert balance_report.balance_id == BALANCE_ID
+    assert balance_report.time_zone == "Europe/Amsterdam"
+    assert balance_report.date_from == "2021-01-01"
+    assert balance_report.date_until == "2021-01-31"
+    assert balance_report.grouping == "transaction-categories"
+    assert balance_report.totals == {"some_data": "some_data"}
+
+
+def test_get_primary_balance_report(client, response):
+    """Get the primary balance report."""
+    response.get(f"https://api.mollie.com/v2/balances/{BALANCE_ID}", "balance_single")
+    response.get(f"https://api.mollie.com/v2/balances/{BALANCE_ID}/primary/report", "balance_report_single")
+
+    balance = client.balances.get(BALANCE_ID)
+    balance_report = balance.report.get(primary=True)
+    assert isinstance(balance_report, BalanceReport)
+
+    assert balance_report.resource == "balance-report"
+    assert balance_report.balance_id == BALANCE_ID
+    assert balance_report.time_zone == "Europe/Amsterdam"
+    assert balance_report.date_from == "2021-01-01"
+    assert balance_report.date_until == "2021-01-31"
+    assert balance_report.grouping == "transaction-categories"
+    assert balance_report.totals == {"some_data": "some_data"}
+
+
+def test_get_balance_transaction(client, response):
```

Comment:
You are getting multiple transactions, so it should be `test_get_balance_transactions`